Sources/PushNotifications.swift

```diff
@@ -381,3 +387,14 @@ import Foundation
     }
     #endif
 }
+
+// The delegate of a push notifications object must adopt the InterestsChangedDelegate protocol.
+@objc public protocol InterestsChangedDelegate: class {
+    /**
+     Tells the delegate that the interests list has changed.
+
+     - Parameter interests: List of interests.
+     */
+    /// - Tag: interestsSetChanged
+    func interestsSetChanged(interests: Array<String>)
```

Comment:
Isn't ios names more like `didInterestSetChanged`?

Reply:
It depends.
It could be something like `interestsSetDidChange`.

Some examples [here](https://developer.apple.com/documentation/uikit/uitextviewdelegate).
There's one example in the [naming conventions document](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html#//apple_ref/doc/uid/20001282-1001803-BCIDAIJE):

Example: `- (void)browserDidScroll:(NSBrowser *)sender;`

Should we go with `interestsSetDidChange`?

WDYT?

Reply:
`interestsSetDidChange` sounds good to me.